lib/rides/pages/search_ride_page.dart

```diff
@@ -1,21 +1,206 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_app/rides/pages/search_deals_page.dart';
+import 'package:flutter_app/util/search/address_search_field.dart';
+import 'package:flutter_app/util/search/address_suggestion.dart';
+import 'package:flutter_app/util/submit_button.dart';
+import 'package:intl/intl.dart';
 
 class SearchRidePage extends StatefulWidget {
-  const SearchRidePage({super.key});
+  const SearchRidePage({Key? key}) : super(key: key);
 
   @override
   State<SearchRidePage> createState() => _SearchRidePageState();
 }
 
+
 class _SearchRidePageState extends State<SearchRidePage> {
+
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  final TextEditingController _startController = TextEditingController();
+  final TextEditingController _destinationController = TextEditingController();
+
+  AddressSuggestion? _startSuggestion;
+  AddressSuggestion? _destinationSuggestion;
+
+  final _dateController = TextEditingController();
+  final _timeController = TextEditingController();
+  late final DateTime _firstDate;
+  late DateTime _selectedDate;
+  late int _dropdownValue;
+
+  final List<int> list = List.generate(10, (index) => index + 1);
+
   @override
+  initState() {
+    super.initState();
+    _firstDate = DateTime.now();
+    _selectedDate = DateTime.now();
+    _dateController.text = _formatDate(_selectedDate);
```

Comment:
```suggestion
    _dateController.text = _formatDate(_selectedDate);
    _timeController.text = _formatTime(_selectedDate);
```
then we have a default value for the time too.